### ISSUE TITLE
Crave TV | crave.ca

### ISSUE THREAD

**Issue by jambgaming** [PreMiD/Presences#2761]
**Prerequisites and essential questions** 
- [x] Is it a popular site?
- [x] Is the website older than 2 months? 
- [x] Is the site locked to a specific country/region? Canada Only
- [x] Is the site a paid service? (e.g. Netflix, Hulu)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?** 
Benjamin#0004
**What's the name of the service?** 
www.crave.ca | Crave TV
**What should the Presence display?** 
Basically the same as the Netflix Presence
 Example:
 Crave TV
                                                                                  Doctor Who
                                                                                 S1:E1 Rose 
**If possible, please provide a logo for the service (512x512 minimum)** 
![Webp net-resizeimage](https://user-images.githubusercontent.com/47050555/103809088-3c884c00-5016-11eb-98df-d8db3235d04c.png)
